index.bs

```diff
@@ -560,76 +586,535 @@ message? -->
+    [HtmlMediaElement.audioTracks](https://html.spec.whatwg.org/multipage/media.html#dom-media-audiotracks)
+    for more details.
+
+: select-video-track-id
+:: Select the given video track by ID and unselect all other video tracks.  See
+    [HtmlMediaElement.videoTracks](https://html.spec.whatwg.org/multipage/media.html#dom-media-videotracks)
+    for more details.
+
+: added-text-tracks
+:: Add text tracks with the given kinds, labels, and languages. See
+    [HtmlMediaElement.addTextTrack](https://html.spec.whatwg.org/multipage/media.html#dom-media-addtexttrack)
+    for more details.  This is optional for the receiver to support and if not
+    supported, the receiver will behave as though it were never set.
+    
+: changed-text-tracks 
+:: Change text tracks by ID.  All other text tracks are left
+    unchanged.  Set the mode, add cues, and remove cues by id.  See
+    [HtmlMediaElement.textTracks](https://html.spec.whatwg.org/multipage/media.html#dom-media-texttracks)
+    for more details.  Note that future specifications or extensions to this
+    specifications are expected to add values to the added text-track-cue
+    values (such as text size, alighment, position, etc).  Adding and removing
+    cues is optional for the receiver to support and if not supported, the
+    receiver will behave as though no cues were added or removed (both adding
+    and removing are indicated via the support for "added-cues").
+
+
+
+The states sent by the receiver include the following individual state values,
+each of which is optional.  This allows the receiver to update the controller
+about one state value or many state values at once without having to specify all
+state values every time.  A non-present state value indicates the state has not
+changed.
+
+: supports
+:: The controls the receiver supports.  These may differ for different [=media
+    resource=]s and should not changes unless the [=media resource=] changes.  If
+    not present in the intial state in the remote-playback-start-response
+    message, support for nothing is assumed.
+
+: source
+:: The current [=media resource=] URL. See
+    [HtmlMediaElement.currentSrc](https://html.spec.whatwg.org/multipage/media.html#dom-media-currentsrc).
+    Must be present in the intial state in the remote-playback-start-response
+    message.
+
+: loading
+:: The state of network activity for loading the [=media resource=].  See
+    [HtmlMediaElement.networkState](https://html.spec.whatwg.org/multipage/media.html#dom-media-networkstate).
+    If not present in the intial state in the remote-playback-start-response
+    message, it is assumed to be empty.
+
+: loaded
+:: The state of the loaded media (whether enough is loaded to play).  See
+    [HtmlMediaElement.readyState](https://html.spec.whatwg.org/multipage/media.html#dom-media-readystate).
+    If not present in the intial state in the remote-playback-start-response
+    message, it is assumed to be nothing.
```

Comment:
Is there a difference between "empty" (line 952) and nothing here?

Reply:
NETWORK_EMPTY and HAVE_NOTHING are not the same in https://html.spec.whatwg.org/multipage/media.html#dom-media-readystate